spec/param.json

```diff
@@ -28,6 +28,10 @@
             "description": "An optional boolean signifying the parameter should be obscured when displayed.",
             "type": "boolean",
             "default": false
-        }
+        },
+        "param": [
```

Comment:
> I'm a tad confused, shouldn't this be an array with [ rather than {?

I know it's confusing. We're actually describing the param object using a json obj so we're saying it's an array type.